src/Lib/Managers/ItemsManager.vala

```diff
@@ -202,7 +207,11 @@ public class Akira.Lib.Managers.ItemsManager : Object {
 
     private void udpate_default_values () {
         border_size = settings.set_border ? settings.border_size : 0.0;
-        border_color = settings.set_border ? settings.border_color: "";
-        fill_color = settings.fill_color;
+
+        string border_color_str = settings.set_border ? settings.border_color : DEFAUL_BORDER_COLOR;
```

Comment:
There's no need for a default border color.
If the `settings.set_border` is `false` it means the user doesn't want a default border color.
And if we need to have a default color for the model, let's use the `settings.border_color` without the necessity of defining another constant variable.